README.md

```diff
@@ -2,30 +2,69 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Add private instance variables to a Java class
+- Instantiate 2 class instances and assign values to the instance variables
+- Print object state
+- Create and use a constant variable
 
-## Introduction
+## Instructions
 
-A 1-2 sentence summary of what will be covered.
+The starter project contains a class named `Employee` with an empty `main` method.
 
-## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+## Task #1 - Add private instance variables and create 2 `Employee` objects
+
+1. Edit the `Employee` class to add 2 private instance variables named `name` and `hoursWorked`.
+   The `name` is a string, while `hoursWorked` is an int.
+   -Make sure you use the `private` accessor keyword for each instance variable.
```

Comment:
Is the `-` needed right before the "Make"? Or is this supposed to be on a new line instead?

Reply:
typo fixed.